tools/ci_build/github/azure-pipelines/py-packaging-pipeline.yml

```diff
@@ -14,7 +14,12 @@ parameters:
   type: boolean
   default: true
 
-- name: enable_windows_gpu
+- name: enable_windows_cuda
+  displayName: 'Whether Windows GPU package is built.'
```

Comment:
The disable name is same as enable_windows_dml. Suggest to change both names to be distinguishable.